src/applications/personalization/profile/tests/e2e/direct-deposit/gating.cypress.spec.js

```diff
@@ -144,6 +148,36 @@ describe('Direct Deposit', () => {
     cy.findByTestId('not-all-data-available-error').should('exist');
     cy.findByText(/something went wrong/i).should('exist');
   });
+  it('should not be blocked if `GET payment_information` fails but they have DD4EDU set up', () => {
```

Comment:
Skeleton tests that currently do pass. I'll flesh these out with more assertions to look for the appropriate error content when we figure out what that content actually is.